src/main/scala/firrtl/passes/memlib/MemConf.scala

```diff
@@ -50,7 +50,7 @@ object MemConf {
 
   def fromString(s: String): Seq[MemConf] = {
     s.split("\n").toSeq.map(_ match {
-      case MemConf.regex(name, depth, width, ports, maskGran) => Some(MemConf(name, depth.toInt, width.toInt, MemPort.fromString(ports), Option(maskGran).map(_.toInt)))
+      case MemConf.regex(name, depth, width, ports, maskGran) => Some(MemConf(name, BigInt(depth), width.toInt, MemPort.fromString(ports), Option(maskGran).map(_.toInt)))
```

Comment:
Isn't `depth` already a `BigInt`? I think you only need to remove the `.toInt` without adding anything else.

Reply:
`depth` in this context is a `String`